source/index.html.md.erb

```diff
@@ -76,6 +76,13 @@ For example, using the curl command line utility tool:
 curl https://www.gov.uk/api/content/take-pet-abroad
 ```
 
+Or, you can use the [govuk-browser-extension](https://github.com/alphagov/govuk-browser-extension):
+
+- Visit the page you want to see content information on
+- Click the GOV.UK extension button in your toolbar
+- Click `Content item (JSON)`
+- You'll be automatically redirected to the content API view
+
```

Comment:
Ah fair enough, I didn't know these docs were any more geared towards the public than our normal dev docs. Think I should change it to something like `Or, you can use the internal [govuk-browser-extension] (note that this is unsupported).`?

Reply:
Yeah, I think something that makes it clear that we don't support it is fine.